manifests/d/DevToys-app/DevToys/2.0-preview.2/DevToys-app.DevToys.locale.en-US.yaml

```diff
@@ -3,7 +3,7 @@
 
 PackageIdentifier: DevToys-app.DevToys
 PackageName: DevToys
-PackageVersion: 2.0.2
+PackageVersion: 2.0-preview.2
 PackageLocale: en-US
 Publisher: DevToys-app
```

Comment:
Windows ARP (settings > apps > installed apps) shows `DevToys` not `DevToys-app` for Publisher